pyi.py

```diff
@@ -1621,6 +1623,26 @@ def _check_if_expression(self, node: ast.expr) -> None:
         else:
             self.error(node, Y002)
 
+    def _check_for_Y066_violations(self, node: ast.If) -> None:
+        test = node.test
+        if not isinstance(test, ast.Compare):
+            return
+
+        left = test.left
+        op = test.ops[0]
+        if (
+            isinstance(left, ast.Attribute)
+            and _is_name(left.value, "sys")
+            and left.attr == "version_info"
+            and isinstance(op, ast.Lt)  # sys.version_info < ...
+            and node.orelse
+            and not (
+                len(node.orelse) == 1 and isinstance(node.orelse[0], ast.If)
+            )  # elif statement
```

Comment:
I don't _think_ you currently have a test where this condition holds true. Could you add one, to make sure that we don't emit a false positive on code like this (which _doesn't_ have an `else` statement at the end)?

```py
class Foo:
    if sys.version_info < (3, 8):
        fields: list[int]
    elif sys.version_info < (3, 9):
        fields: list[int | str]
```